From 9e96d2b39648a2dbb1f6f6b8cca2ffce4dcd24b9 Mon Sep 17 00:00:00 2001
From: KancyJoe <kancy2333@outlook.com>
Date: Wed, 17 Dec 2025 07:58:07 +0000
Subject: [PATCH] add back hyp log disable function for kailua linux
 compatibility.

---
 QcomModulePkg/Application/LinuxLoader/LinuxLoader.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/QcomModulePkg/Application/LinuxLoader/LinuxLoader.c b/QcomModulePkg/Application/LinuxLoader/LinuxLoader.c
index d579d2b503..a9bd7650da 100644
--- a/QcomModulePkg/Application/LinuxLoader/LinuxLoader.c
+++ b/QcomModulePkg/Application/LinuxLoader/LinuxLoader.c
@@ -286,6 +286,10 @@ LinuxLoaderEntry (IN EFI_HANDLE ImageHandle, IN EFI_SYSTEM_TABLE *SystemTable)
 
   BootStatsSetTimeStamp (BS_BL_START);
 
+  if (IsVmEnabled ()) {
+    DisableHypUartUsageForLogging ();
+  }
+
   /* Check if memory card is present; goto flashless if not */
   Status = gBS->LocateProtocol (&gEfiMemCardInfoProtocolGuid, NULL,
                                   (VOID **)&CardInfo);
-- 
2.53.0

